README.md

```diff
@@ -19,3 +32,53 @@ To run the app in production mode, run:
 `npm run dist` will compile and minify the Javascript and SCSS.
 `NODE_ENV=production` is an environment variable that tells the app to run in production mode,
 and to not start the webpack dev server.
+
+## Contributing
+
+### Git Workflow
+
+ - Cut feature branch from `development`
+ - Create PR against `development`
+ - After approval, author merges feature into `development`
+ - Merge `development` > `qa`
+ - Merge `qa` > `master` (production)
+
+## Troubleshooting
+
+### Stacktraces in logs
```

Comment:
Thanks for writing this. This hasn't been a problem in newer apps because of the updated packages.